tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -99,6 +101,8 @@ public Span RootSpan
         /// </summary>
         internal IastRequestContext IastRequestContext => _iastRequestContext;
 
+        internal List<object> WafSecurityEvents { get; set; }
```

Comment:
I wish this file was already nullable enabled 🙁

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/commit/937873794fa4d47cba80d594032913fb2903e768